taperole/hosts

```diff
@@ -0,0 +1,9 @@
+[production]
+162.243.113.144 be_app_env=production be_app_branch=tape
```

Comment:
@cwjenkins Yes - I wanted to do it on a branch since I've never used Tape, just in case I screwed something up :)